cherrypy/lib/static.py

```diff
@@ -28,6 +29,31 @@ def _setup_mimetypes():
 _setup_mimetypes()
 
 
+def _make_content_disposition(disposition, file_name):
+    """Create HTTP header for downloading a file with a UTF-8 filename.
+
+    This function implements the recommendations of :rfc:`6266#appendix-D`.
+    See this and related answers: https://stackoverflow.com/a/8996249/2173868.
+
+    As normalization algorithm for `unicodedata` is used composed form (NFC and NKFC)
```

Comment:
I don't think this should go to the docstring but to a comment. Docstrings should only contain high-level usage implications information while comments answer to the question why a certain technical decision was made.

Reply:
(also this exceeds the line limit as you can see from the linter reports which has to be fixed for use to be able to merge this)

Reply:
I moved it below as code comment

Reply:
Good, but it still needs to pass the linter checks.